src/nfa/operations.cc

```diff
@@ -692,6 +720,7 @@ Nfa Mata::Nfa::minimize(
 	return algo(aut);
 }
 
+//could be a method that does it in-place (modifies the object on which it is called)?
```

Comment:
We agreed that functions should return new instance and methods should compute inplace, if at all possible. Therefore, this function should remain a function.

Furthermore, we should add an in-place union of NFAs, replacing the instance the function was called on. The method should be named `join_with(Nfa other)`, `compute_union_with(Nfa other)` or something similar.

Reply:
maybe unite_with, or something completely different indicating that we are adding something to the existing automaton.

Reply:
`unite_with()` sounds good to me, too.

Reply:
Resolved in #412, as per the suggestion in #354.